content/en/numpy-code-of-conduct.md

```diff
@@ -0,0 +1,83 @@
+<h2>NumPy Code of Conduct</h2>
```

Comment:
Same comment as above, use:
```
---
title: NumPy Code of Conduct
sidebar: false
---

```

Reply:
Also, can you rename the file to simply `code-of-conduct.md`? Looks better, and will make the link in the footer work.

Reply:
Done.